packages/reputation-algorithms/docs/README.md

```
@@ -0,0 +1,120 @@
+**@reputo/reputation-algorithms v0.0.0**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)